cpu/sam0_common/periph/uart.c

```diff
@@ -98,6 +100,10 @@ int uart_init(uart_t uart, uint32_t baudrate, uart_rx_cb_t rx_cb, void *arg)
         NVIC_EnableIRQ(SERCOM0_IRQn + sercom_id(dev(uart)));
         dev(uart)->CTRLB.reg |= SERCOM_USART_CTRLB_RXEN;
         dev(uart)->INTENSET.reg |= SERCOM_USART_INTENSET_RXC;
+        /* set wakeup receive from sleep if enabled */
+        if ((uart_config[uart].flags & UART_WAKEUP) == UART_WAKEUP) {
```

Comment:
same here, simplify to: `if (uart_config[uart].flags & UART_WAKEUP) {`

Reply:
Fixed